lightning/src/ln/channel.rs

```diff
@@ -4780,7 +4783,7 @@ impl<Signer: Sign> Channel<Signer> {
 			// We got a reorg but not enough to trigger a force close, just ignore.
 			false
 		} else {
-			if self.channel_state < ChannelState::ChannelFunded as u32 {
+			if self.funding_tx_confirmation_height != 0 && self.channel_state < ChannelState::ChannelFunded as u32 {
```

Comment:
Minor: comment could be stronger here it's preventing to hit the panic when we receive a `transactions_confirmed()` in the 0conf case as raised in pre-review.